src/Symfony/Component/HttpFoundation/RequestMatcher/IsJsonRequestMatcher.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\RequestMatcher;
+
+use Symfony\Component\HttpFoundation\Exception\JsonException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+
+/**
+ * Checks the Request content is valid JSON.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class IsJsonRequestMatcher implements RequestMatcherInterface
```

Comment:
Looks like a new one, how can it be used in the chain from the config?

Reply:
the SecurityBundle access control rules indeed don't have a shortcut for using it. So in case you want to use it (but a security rule applying only for valid json payload looks like a weird use case), you would need to use the config setting allowing to use a custom request matcher service for that rule.